megamek/src/megamek/common/Entity.java

```diff
@@ -217,6 +217,9 @@
     protected boolean useManualBV = false;
     protected int manualBV = -1;
 
+    protected int initialBV = -1;
+    protected boolean initialBVSet = false;
```

Comment:
I don't see any reason for the `initialBVSet` field, when you can test whether it has been set with `initialBV < 0`.

Since `initialBV` is only used in the client and does not change, it could be marked `transient` to keep from sending the value back and forth between the client and the server each time the `Entity` is updated.